tests/phpunit/CRM/Export/BAO/ExportTest.php

```diff
@@ -1756,6 +2063,7 @@ protected function getBasicSqlColumnDefinition($isContactExport) {
       'country' => 'country varchar(64)',
       'phone' => 'phone varchar(32)',
       'phone_ext' => 'phone_ext varchar(16)',
+      'phone_type_id' => 'phone_type_id varchar(16)',
```

Comment:
@eileenmcnaughton is the change here because it is now able to use the field definition and not fall back to a default length of 255?

Reply:
@seamuslee001 yep - it's an unintended consequence but one which I think is OK since many other fields are given 16 char. I want to look at that side of things more once I get to the end of the cleanup